tests/test.js

```diff
@@ -24,7 +24,7 @@ class TimeEntriesPage {
     await this.page.locator('label:has-text("to:")').fill(to);
   }
 
-  async addEntry(year, month, date, hours, description) {
+  async addEntry(year, month, date, hours, description, useEnter = false) {
```

Comment:
I think you added the parameter to the wrong method head. Move this to the "editEntry" method.

Reply:
You are absolutely correct. Merge conflict 🚨